app/scripts/modules/core/src/pipeline/config/stages/gremlin/GremlinStageConfig.tsx

```diff
@@ -0,0 +1,204 @@
+import * as React from 'react';
+
+import { API } from '@spinnaker/core';
```

Comment:
```suggestion
import { API } from 'core/api/ApiService';
```
It's a long story, but we don't want to import using the module packages within those same module packages